src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -473,6 +473,13 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                         replacementLength = 0;
                         break;
 
+                    case TokenKind.Semi:
```

Comment:
Does adding this case affect `get-something; e<TAB>` - in other words, no hashtable, just simple command completion?

Reply:
No, I don't see the side affect.

Reply:
I re-check `get-something; e<Tab>` - in Windows PowerShell 5.1 and in PowerShell Core 6.0 simple command completion works - nothing new added in the PR.